content/en/donate.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Donate to the Medley Interlisp Project
+weight: 20
+type: docs
+aliases:
+ - /donate
+---
+
+InterlispOrg Inc, the non-profit which runs the Medley Interlisp Project, uses donations to pay for online services, web hosting; larger donations for hiring consultants and contract workers to help with various tasks.
+
+## How to donate
+
+You can donate through the following methods:
+
+<form action="https://www.paypal.com/donate" method="post" target="_top">
```

Comment:
Move this to be below line 9 - so, we have the text about being a non-profit, followed immediately be the donate button.